test/time_unittest.cc

```diff
@@ -156,4 +156,26 @@ TEST(TimeTest, FromStringScientificSeconds) {
   );
 }
 
+TEST(TimeTest, SecondsSinceEpoch) {
+  const time_point t =
+      time::rfc3339::FromString("2017-07-14T02:40:00Z");
```

Comment:
This is a bit unintuitive. Would it make sense to use something like `const time_point t = time_point() + seconds(1500000000)` instead?

Reply:
Done.